<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1650</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1650</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1650</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="addonConfig">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>60</y>
      <width>281</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Add-On Configuration</string>
    </property>
    <widget class="QGroupBox" name="ballConfig">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>241</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Ball Configuration</string>
     </property>
     <widget class="QLabel" name="velocity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ball Velocity</string>
      </property>
     </widget>
     <widget class="QLabel" name="direction">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ball Direction</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="velocityInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="directionInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addBall">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Ball</string>
      </property>
     </widget>
     <widget class="QLabel" name="locationX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ball Location (X)</string>
      </property>
     </widget>
     <widget class="QLabel" name="locationY">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ball Location (Y)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="yInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="wallConfig">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>241</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Wall Configuration</string>
     </property>
     <widget class="QLabel" name="x1Location">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wall Location 1 (X1)</string>
      </property>
     </widget>
     <widget class="QLabel" name="Rotation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wall Rotation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Wall</string>
      </property>
     </widget>
     <widget class="QLabel" name="y1Location">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wall Location1  (Y1)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="x2Location">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wall Location 2 (X2)</string>
      </property>
     </widget>
     <widget class="QLabel" name="y2Location">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wall Location 2 (Y2)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="field">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fpsCounterLabel">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS: XX </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
